packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -245,16 +245,17 @@ class FlatTableState<T> extends State<FlatTable<T>> with AutoDisposeMixin {
     );
   }
 
-  Widget _buildRow({
+  Widget? _buildRow({
```

Comment:
this doesn't feel safe. We shouldn't be returning null for a row in the table. This looks like a bandaid for some exception rather than a fix for the root cause. `index` should never be greater than `data.length` so whatever is causing that to happen is the bug we need to fix.

Reply:
As the flutter's TableView is on the way, are you sure we want to heavily invest into maintaining our table?

This is caused by shrinking the dataset:
1. Scroll to the end in the table of retaining paths
2. Switch to other class or snapshot and scroll to the end again
3. Eventually you will get to smaller list of the retaining paths, that will cause the error

Reply:
removed nullability as discussed offline